unit-test/background/tracker-utils-tests.js

```diff
@@ -627,3 +627,39 @@ describe('Tracker Utilities', () => {
         tdsStorage.ClickToLoadConfig = originalConfig
     })
 })
+
+describe('trackerutils.isFirstPartyByEntity()', () => {
+    let timer = Date.now()
+    const jump = 1000 * 60 * 31 // slightly more than cache timeout
+
+    beforeAll(() => {
+        spyOn(settings, 'getSetting')
+        tdsStorageStub.stub()
+
+        // Make sure we don't use any list caches for these tests
+        spyOn(Date, 'now').and.callFake(function () {
+            // Cache may be updated on each run, so keep jumping the time forward for each call
+            timer += jump
+            return timer
+        })
+
+        return tdsStorage.getLists()
+            .then(lists => {
+                return tds.setLists(lists)
+            })
+    })
+
+    const firstPartyTests = [
+        { a: 'http://google-analytics.com', b: 'http://google.com', expected: true },
+        { a: 'disqus.com', b: 'google.com', expected: false }
```

Comment:
These should be full URLs, not just hostnames, right?

Reply:
I'll make them urls for consistency but the utility function doesn't care.